landing.css

```diff
@@ -6,21 +6,21 @@
     padding: 0;
     box-sizing: border-box;
 }
+
 body{
     display: flex;
     /* flex-wrap: wrap; */
     flex-direction: column;
     font-family: sans-serif;
-    color: var(--primary-color);
 }
+
 .header{
     display: flex;
     flex-wrap: wrap;
     box-shadow: 0px 15px 10px -15px #ccc;    
    padding: 1.5rem;
    background-color: #f1f5f9;
```

Comment:
Code indentation looks a bit off here. You could correct this.

Reply:
okk